src/Oceananigans.jl

```diff
@@ -49,7 +49,7 @@ export
 
     # Model output writers
     Checkpointer, restore_from_checkpoint, read_output,
-    JLD2OutputWriter, FieldOutput, FieldOutputs,
+    JLD2OutputWriter, NetCDFOutputWriter, FieldOutput, FieldOutputs,

```

Comment:
?

Reply:
Yeah this was an issue after I merged master in, I guess we forgot to export it in PR #433.

Bad practice on my part to include changes not relevant to reversing the k index.